### ISSUE TITLE
Crash with General Lander

### ISSUE THREAD

**Issue by RoseQuartz-z** [MJRLegends/ExtraPlanets#243]
### ---Issue Report---

**Have you checked Closed Issues (if applicable & PLEASE CHECK BEFORE POSTING):** Yes

### Description of Issue
Every time I attempt to travel to a planet or moon that is by default included in Galacticraft, I will be stuck in a loading screen for several minutes until the game crashes. When I go back into the map, I find that I have teleported to Oberon or Callisto, often inside of an impact crater at my feet. When trying to travel to the "vanilla" planets using a launch controller by setting the corresponding launch ID, it will pull up the map of the solar system regardless, and when clicking on the planet, the problem will repeat itself.

### Steps to Reproduce Issue
1. Boot up game
2. Fly literally anywhere

**Crash log/Log (if applicable):** https://pastebin.com/6rTHjpmh

**Screenshots (if applicable):** None

**Version of Mod using (PLEASE DONT USE 'Latest' OR 'Newest'):** ExtraPlanets-1.12.2-0.2.3.jar

**Where did this bug happen:** Single player. 

### Additional Information
I am playing on a Biomes O' Plenty World that has been modified extensively using Worldpainter. When by itself, Galacticraft works perfectly, and when I have Extra Planets installed, it works just as well. They just don't seem to mix. 

Thank you very much for your help! I love the mod (at least, what I can get to work!)
                        -Jaelyn

**Comment by MJRLegends** [comment]
This is a MJRLegendsLib issue and a update is coming out now for it :P

**Comment by MJRLegends** [comment]
Thank you for the bug report too

**Comment by RoseQuartz-z** [comment]
Thank you!

**Comment by MJRLegends** [comment]
No problem

**Comment by RoseQuartz-z** [comment]
Hey, so, with the new update, I can now access all planets, (including Mars
and Venus), but there seems to be bugs with the moons. I spawned on the
Earth's moon, and saw alerts telling me that I was under high pressure. The
sky was also red, and the sun looked way farther away than it should have
looked. In addition, the Journeymap mod told me that there were several
biomes in the landscape, all named after other moons (Titan, Triton, etc.).
I shrugged it off, thinking it wasn't much of a problem. I then tested out
Mars and Neptune, and there were absolutely no issues whatsoever. I decided
to attempt returning to the moon, when I ran into the infinite loading
screen again - after about seven minutes, I closed it. The next day, I
booted up Minecraft again, and it spawned me in a strange landscape made of
Oberon stone, but it was marked by Journeymap as "Phobos". Attached is a
screenshot of the strange Oberon/Phobos hybrid.
 [image: Inline image 1]
